pyincore_data/utils/nsiutil.py

```diff
@@ -245,6 +398,8 @@ def assign_hazus_specific_structure_type(
         gdf["year_built"] = year_built
         gdf["dgn_lvl"] = dgn_lvl
         gdf["exact_match"] = exact_match
+        gdf["f_arch"] = f_arch
```

Comment:
The column names should be "arch_wind" and "arch_flood" following the Galveston building example.

Reply:
the field names are change to arch_wind and arch_flood